qiita_db/environment_manager.py

```diff
@@ -436,5 +436,6 @@ def patch(patches_dir=PATCHES_DIR, verbose=False, test=False):
         # for the test Study (1) so alot of the tests actually expect this.
```

Comment:
Outside the scope of this change, but if you can change 'alot' -> 'a lot', that would be awesome ty.

Reply:
k